AnimatedWordCloud/Animator/TimelapseWordVector.py

```diff
@@ -22,11 +23,15 @@ def __init__(self) -> None:
         Prepare empty data
         """
 
-        # use heap to easily get the rankings
-        self._word_heap: List[Tuple[str, float]] = []
+        # use bisect to easily get the rankings
+        # to order by weight,
+        #   the weight must be the first element of the tuple
+        #   and negate the weight to get the descending order
+        # but the output must be the word first
+        self._word_bisect: List[Tuple[float, str]] = []
```

Comment:
It is better to specify that the element should be added with WordVector.add

For example:
instance = WordVector()
instance._word_bisect.append((-2, "test5"))
instance._word_bisect.append((-3, "test5"))
instance.add("test", 1)
instance.add("test2", 4)
instance.add("test3", 3)
instance.add("test4", 2)

print(instance._word_bisect)
[(-4, 'test2'),
 (-3, 'test3'),
 (-2, 'test4'),
 (-2, 'test5'),
 (-3, 'test5'),
 (-1, 'test')]

Reply:
Added warning comment